process/block/metablock_test.go

```diff
@@ -538,6 +538,96 @@ func TestMetaProcessor_ProcessBlockWithErrOnVerifyStateRootCallShouldRevertState
 	assert.True(t, wasCalled)
 }
 
+//------- processBlockHeader
+
+func TestMetaProcessor_ProcessBlockHeaderShouldPass(t *testing.T) {
+	t.Parallel()
+
+	mdp := initMetaDataPool()
+	accounts := &mock.AccountsStub{}
+	accounts.RevertToSnapshotCalled = func(snapshot int) error {
+		return nil
+	}
+	mp, _ := blproc.NewMetaProcessor(
```

Comment:
the following sequence seems to be used in almost every single test:

```go
	mdp := initMetaDataPool()
	accounts := &mock.AccountsStub{}
	accounts.RevertToSnapshotCalled = func(snapshot int) error {
		return nil
	}
	mp, _ := blproc.NewMetaProcessor(
		&mock.ServiceContainerMock{},
		accounts,
		mdp,
		&mock.ForkDetectorMock{},
		mock.NewMultiShardsCoordinatorMock(3),
		&mock.HasherStub{},
		&mock.MarshalizerMock{},
		&mock.ChainStorerMock{},
		createGenesisBlocks(mock.NewMultiShardsCoordinatorMock(3)),
		&mock.RequestHandlerMock{},
	)
```

so I think this can be extracted into a function and used instead in all tests that make use of this code sequence.

Reply:
done